templates/web/common/site_layout.tt.html

```diff
@@ -19,6 +19,7 @@
     <link rel="stylesheet" href="[% static_subdomain %]/css/dist/app-[% edq(lang('text_direction')) %].css?v=[% css_timestamp %]" data-base-layout="true">
     <link rel="stylesheet" href="[% static_subdomain %]/css/dist/jqueryui/themes/base/jquery-ui.css" data-base-layout="true">
     <link rel="stylesheet" href="[% static_subdomain %]/css/dist/select2.min.css">
+	<link rel="stylesheet" href="[% static_subdomain %]/css/dist/custom-tagify.css" data-base-layout="true">
```

Comment:
We only really need Tagify in https://github.com/openfoodfacts/openfoodfacts-server/blob/24e957738c1eccd4341e21fd2411069aaa933c4c/cgi/product_multilingual.pl#L777, so this probably needs to be removed, and the import in `cgi/product_multilingual.pl` should be updated.